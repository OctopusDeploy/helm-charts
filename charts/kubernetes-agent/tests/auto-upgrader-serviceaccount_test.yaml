suite: "pod permissions"
templates:
- templates/auto-upgrader-serviceaccount.yaml
tests:
- it: "should match snapshot"
  asserts:
  - matchSnapshot: {}

- it: "should have fixed service account name"
  set: 
    agent:
      targetName: "test"
    nameOverride: "test_test"
  asserts:
  - equal:
      path: "metadata.name"
      value: "octopus-agent-auto-upgrader"
    
