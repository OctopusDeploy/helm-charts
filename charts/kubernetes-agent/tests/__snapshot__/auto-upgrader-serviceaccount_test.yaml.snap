should match snapshot:
  1: |
    apiVersion: v1
    automountServiceAccountToken: true
    kind: ServiceAccount
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: octopus-agent
        app.kubernetes.io/version: 8.3.3034
        helm.sh/chart: kubernetes-agent-2.19.0
      name: octopus-agent-auto-upgrader
      namespace: NAMESPACE
